src/test/regress/sql/local_table_join.sql

```diff
@@ -0,0 +1,389 @@
+CREATE SCHEMA local_table_join;
```

Comment:
my wip test file, most of which are already covered here but I'm still building further: https://gist.github.com/onderkalaci/939eea594e005de0cbebf31db2c5109b